src/makeRobot.js

```diff
@@ -39,6 +39,69 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(steps = 1) {
+      if (steps < 0) {
+        return this;
+      }
+
+      this.coords.y += 1 * steps;
+
+      return this;
```

Comment:
```suggestion
    goForward(steps = 1) {
      if (steps > 0) {
         this.coords.y += steps;  
      }

      return this;
```
You will return this anyway. Just do nothing if steps < 1, 
Also, why do you need to multiply steps with 1?